src/python/dxpy/dx_extract_utils/ExpressionInputsValidator.py

```diff
@@ -62,6 +59,11 @@ def end_arguments_check(self, end_arg):
                 )
             )
 
+    def invalid_combinations_check(self, invalid_combinations_info):
+        for e in invalid_combinations_info:
+            if invalid_combinations_info[e]["invalid_combination"]:
```

Comment:
```
if e["invalid_combination"]:
```
.. and below ..
```
e["error_message"]
```

Reply:
This doesn't really work cause the for is taking the variable as the strings ("json_help_unique", "retrieve_expression_json" and "empty_json")